src/Nucleation.jl

```diff
@@ -0,0 +1,94 @@
+module Nucleation
+
+"""
+    nucleation_timestep(rh, temp, so4)
+
+ - `rh` - Relative humidity, expressed as a percentage between 0 and 1
+ - `temp` - Temperature (K)
+ - `so4` - Total gas phase concentration of sulfuric acid (cm^-3)
+
+Returns the nucleation rate for a single timestep under the given conditions.
+All newly formed particles will be placed into the Aitken mode.
+The initial nucleation rate comes from Vehkamaki et al, 2002 doi:10.1029/2002JD002184
+Adjustment factor comes from Kerminen and Kulmala, 2002 doi:10.1029/2002JD002184
+
+# Examples
+```julia-repl
+julia> nucleation_timestep(0.55, 236, 1e7)
+162.96173344575243
+```
+"""
+function nucleation_timestep(rh, temp, so4)
```

Comment:
Would it be correct to name this `SO4_nucleation_rate`? Timestep may be confusing, for example when we will use some multi stage timesteppers